src/main/java/org/noteskeeper/server/HelloWorldController.java

```diff
@@ -0,0 +1,21 @@
+package org.noteskeeper.server;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@Controller
```

Comment:
For REST services it is more convenient to use meta-annotation '@RestController'. It encapsulates simple '@Controller' and '@RequestBody' annotations. So you may not specifie last one on methods.